src/systems/userinput/bindings/oculus-touch-user.js

```diff
@@ -793,11 +793,6 @@ export const oculusTouchUserBindings = addSetsToBindings({
     {
       src: { value: leftGripPressed2 },
       dest: { value: leftGripRisingGrab },
-      xform: xforms.rising
-    },
-    {
-      src: { value: leftTriggerPressed2 },
-      dest: { value: leftTriggerRisingGrab },
```

Comment:
these targets didn't seem consumed, any idea on if they are actually needed?

Reply:
Seems safe to remove